pkgs/servers/nosql/mongodb/5.0.nix

```diff
@@ -6,22 +6,24 @@ let
   };
   variants = if stdenv.isLinux then
     {
-      version = "5.0.7";
-      sha256 = "sha256-1PeDBZJNqJXHH/cSh2e+WR0PfS/b7XuJEzkkbrRT/gc=";
+      version = "5.0.18";
+      sha256 = "sha256-tvQkDBwXYRZbIuST49JJ5T9zzYe/4BQ8ul1vUGlXHxI=";
+      patches = [ ./fix-build-with-boost-1.79-5_0-linux.patch ];
     }
-  else
+  else if stdenv.isDarwin then
     {
       version = "5.0.3"; # at least darwin has to stay on 5.0.3 until the SDK used by nixpkgs is bumped to 10.13
       sha256 = "1p9pq0dfd6lynvnz5p1c8dqp4filzrz86j840xwxwx82dm1zl6p0";
-    };
+      patches = [ ./fix-build-with-boost-1.79-5_0-darwin.patch ];
+    }
+  else {};
 in
 buildMongoDB {
   version = variants.version;
   sha256 = variants.sha256;
```

Comment:
```suggestion
  inherit (variants) version sha256;
```
nit

Reply:
not a fan of too much short inherits. this is not long enough to make it more readable this way imho.